### PR TITLE
Add functionality to Briefing Room/Add Scenario

### PR DESCRIPTION
**Pull request by sldfgunslinger2766**
This PR adds the ability to impose flat C-Bill costs to New Missions created in the Briefing Room/Add Scenario Pane for use in Chaos Campaign style games using C-Bills. Changes to the JSpinner allow negative values to be applied in the Add Loot/Edit Loot to simulate costs for Missions that are debited to a Units finances by enabling the appropriate check box in the game resolution. Some minor verbiage changes were added in the Panes and Mission Reports to reflect the changes made.

### CODE REVIEW

File: MekHQ/src/mekhq/campaign/mission/Loot.java [hunk 1]
```diff
@@ -107,11 +107,15 @@ public void clearParts() {
     }
 
     public String getShortDescription() {
-        String desc = getName() + " - ";
+        String desc = getName() + "  ";
         if (cash.isPositive()) {
             desc += cash.toAmountAndSymbolString();
         }
 
+        if (cash.isNegative()) {
```
Comment (NickAragua) [inline MekHQ/src/mekhq/campaign/mission/Loot.java:115]: At this point, you might as well just remove the if blocks and always append cash.toAmountAndSymbolString() to desc (unless it's zero, I guess).

**Review by NickAragua — CHANGES_REQUESTED** [review]
Reasonable idea, just needs a little cleanup.

**Comment by sldfgunslinger2766** [comment]
Wouldn't removing the if blocks also remove the context of cost and payout as well? That is handy for clear record keeping in the reports.

**Review by NickAragua — APPROVED** [review]
Looking good.

**Pull request merged**